### PR TITLE
curations: Add VCS URLs for log4net

### PR DESCRIPTION
**Pull request by sschuberth**
Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: curations/NuGet/_/log4net.yml [hunk 1]
```diff
@@ -2,19 +2,27 @@
...
     vcs:
+      type: Git
+      url: https://github.com/apache/logging-log4net.git
       revision: "rel/2.0.8"
 - id: "NuGet::log4net:2.0.11"
   curations:
     comment: "Set the VCS revision because there are multiple tags matching the version."
     vcs:
+      type: Git
+      url: https://github.com/apache/logging-log4net.git
```
Comment (mnonnenmacher) [inline curations/NuGet/_/log4net.yml:13]: Starting with 2.0.10 repository metadata should be available, compare:
https://nuget.info/packages/log4net/2.0.9
https://nuget.info/packages/log4net/2.0.10

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

**Comment by mnonnenmacher** [comment]
Since your are at it, would you mind adding a revision curation for 2.0.14 for which also an RC tag exists?

File: curations/NuGet/_/log4net.yml (hunk 1 above)
Comment (sschuberth) [reply curations/NuGet/_/log4net.yml:13]: Ah, that nuget.info site is very useful, thanks!

**Comment by sschuberth** [comment]
> Since your are at it, would you mind adding a revision curation for 2.0.14 for which also an RC tag exists?

Do you think we should hard-code some heuristic regarding tag prefixes to work around this?

**Comment by mnonnenmacher** [comment]
> > Since your are at it, would you mind adding a revision curation for 2.0.14 for which also an RC tag exists?
> 
> Do you think we should hard-code some heuristic regarding tag prefixes to work around this?

If you have a good idea how to improve go for it, but I'm a bit hesitant to touch that code.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**